systers_portal/meetup/views.py

```diff
@@ -12,3 +12,17 @@ class MeetupLocationAboutView(MeetupLocationMixin, TemplateView):
 
     def get_meetup_location(self):
         return get_object_or_404(MeetupLocation, slug=self.kwargs['slug'])
+
+
+class MeetupView(MeetupLocationMixin, TemplateView):
```

Comment:
If you change it to `DetailView`, then you can set `slug_field` and you remove the extending of `get_context_data()`. Pretty neat I think ;)